packages/client/hmi-client/src/components/projects/NewProjectCard.vue

```diff
@@ -19,7 +18,7 @@ async function createNewProject() {
 	const author = auth.name ?? '';
 	const project = await ProjectService.create(name.value, description.value, author);
 	if (project) {
-		router.push(`/projects/${project.id}`);
+		logger.info(`Project ${project.title} created`);
```

Comment:
You removed `router.push` logic, is this intentional?

Reply:
no not all.  didn't mean to stage this change.  thx.